.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,14 @@
 {
 	"name": "DBSP",
-	"dockerFile": "Dockerfile",
+	"dockerComposeFile": [
```

Comment:
Is there a way to provide additional arguments to docker compose? If so, we can reuse the existing docker compose files, and simply say "redpanda" as an argument to only start that service.

Reply:
@lalithsuresh devcontainer now employs "Extend docker-compose" feature